pos_remove_payment_methods/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "POS: clear payment lines on screen entry",
+    "summary": "Clears payment lines when re-entering the POS payment screen",
```

Comment:
@ilmir-k, ок?

Reply:
@em230418 @kopeyev 
"name": "POS: clear **the** payment lines on screen entry",
"summary": "Clears **the** payment lines **that have been selected before in case when we return back to the cart and** re-entering the POS payment screen",